tests/sentry/integrations/jira_server/test_webhooks.py

```diff
@@ -56,6 +62,35 @@ def test_post_empty_token(self):
 
         assert resp.status_code == 400
 
+    def test_post_missing_default_identity(self):
+        org_integration = OrganizationIntegration.objects.get(
+            organization_id=self.organization.id,
+            integration_id=self.integration.id,
+        )
+        org_integration.update(default_auth_id=None)
+        org_integration.update(config={"sync_status_reverse": True})
+        payload = {
+            "changelog": {
+                "items": [
+                    {
+                        "from": "10101",
+                        "field": "status",
+                        "fromString": "In Progress",
+                        "to": "10102",
+                        "toString": "Done",
+                        "fieldtype": "jira",
+                        "fieldId": "status",
+                    }
+                ],
+                "id": 12345,
+            },
+            "issue": {"project": {"key": "APP", "id": "10000"}, "key": "APP-1"},
+        }
+        token = self.jwt_token
```

Comment:
Not required but if you want you can replace everything after this with:
```python
self.get_error_response(self.jwt_token, **payload, status_code=400)
```
if you add:
```python
endpoint = "sentry-extensions-jiraserver-issue-updated"
method = "post"
```
at the top of the class.